flyteidl/boilerplate/flyte/golang_support_tools/go.mod

```diff
@@ -5,33 +5,17 @@ go 1.19
 require (
 	github.com/EngHabu/mockery v0.0.0-20220405200825-3f76291311cf
 	github.com/alvaroloes/enumer v1.1.2
-	github.com/flyteorg/flytestdlib v0.4.16
```

Comment:
we build the `pflags` binary beforehand now, so no need to depend on this.

Reply:
we'll remove boilerplate eventually, right